gensim/models/word2vec.py

```diff
@@ -1116,17 +1116,36 @@ def doesnt_match(self, words):
         dists = dot(vectors, mean)
         return sorted(zip(dists, words))[0][1]
 
-    def __getitem__(self, word):
+    @classmethod
```

Comment:
Sorry, Yeah I was so much into the distance tool that I wanted to mimic it in the python version to support phrases. I split my `__getitem__()` function in my initial commit into mean and `__getitem__()`. A new function is not necessary for this one liner `np.mean(model[["office","products"]],axis=0)`, I will remove it in my next commit.